homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -0,0 +1,72 @@
+"""Platform for Husqvarna Automower basic entity."""
+
+import datetime
+import logging
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from homeassistant.util import dt as dt_util
+
+from . import AutomowerDataUpdateCoordinator
+from .const import DOMAIN, HUSQVARNA_URL
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AutomowerEntity(CoordinatorEntity[AutomowerDataUpdateCoordinator]):
+    """Defining the Automower Basic Entity."""
+
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    def __init__(self, coordinator, idx) -> None:
+        """Initialize AutomowerEntity."""
+        super().__init__(coordinator, context=idx)
+        self.idx = idx
+        self.mower = coordinator.session.data["data"][self.idx]
+        mower_attributes = self.get_mower_attributes()
+        self.mower_id = self.mower["id"]
+        self.mower_name = mower_attributes["system"]["name"]
+        self.model_name = mower_attributes["system"]["model"]
+
+        self._available = self.get_mower_attributes()["metadata"]["connected"]
+
+    def get_mower_attributes(self) -> dict:
```

Comment:
```suggestion
    @property
    def mower_attributes(self) -> dict[str, Any]:
```

Reply:
This way it's an property and can be accessed with `self.mower_attributes`

Reply:
I'm only a hobby programmer. I've always had the problem, that I had call the function. So otherwise the values stayed the same all the time. Is it really that simple solution?

Reply:
you can see it as a semi function, you access it as property, but it calls the function but makes code way cleaner :)